cedar-lean/Cli/Main.lean

```diff
@@ -43,5 +43,8 @@ def main (args : List String) : IO Unit :=
       | "validate" =>
         let response := validateDRT request
         IO.println response
-      | _ => printUsage s!"Invalid command `{command}` (expected `authorize` or `validate`)"
+      | "evaluate" =>
+        let response := evaluate request
+        IO.println response
+      | _ => printUsage s!"Invalid command `{command}` (expected `authorize`, `validate`, or `evaluate`)"
```

Comment:
[nit] oxford commas waste electricity

Reply:
🤣

Reply:
I'll humor you, but I like oxford commas

Reply:
I also vote for oxford commas